File: U1/EXAMEN/BanquitoServer_Soap_DotNet_GR01/BanquitoServer_Soap_DotNet_GR01/DTOs/ValidacionCreditoDTO.cs
using System.Runtime.Serialization;

namespace BanquitoServer_Soap_DotNet_GR01.DTOs
{
    /// <summary>
    /// DTO para respuesta de validación de crédito
    /// </summary>
    [DataContract]
    public class ValidacionCreditoDTO
    {
        [DataMember]
        public bool EsValido { get; set; }

        [DataMember]
        public string Mensaje { get; set; }

        [DataMember]
        public string Cedula { get; set; }

        [DataMember]
        public string NombreCompleto { get; set; }
    }
}

File: U2/PROYECTO/04. SERVIDOR/BanquitoServer_Restfull_Java_GR01/src/main/java/ec/edu/monster/DTOs/ValidacionCreditoDTO.java
package ec.edu.monster.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * DTO para respuesta de validación de sujeto de crédito
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ValidacionCreditoDTO {
    private Boolean esValido;
    private String mensaje;
    private String cedula;
    private String nombreCompleto;
}
